shared/modules/utility_module.go

```diff
@@ -62,11 +62,6 @@ type FishermanModule interface {
 	Module
 }
 
-type ServicerModule interface {
-	Module
-	HandleRelay(*coreTypes.Relay) (*coreTypes.RelayResponse, error)
-}
-
 type ValidatorModule interface {
```

Comment:
I realize it is outside the scope of this PR, but this creates an inconsistency between the different protocol actors. 

Can you also move the other ones into independent files under `shared`?

Reply:
Thank you for the review. Fixed.